apps/src/weblab2/Weblab2View.tsx

```diff
@@ -54,8 +54,8 @@ const defaultConfig: ConfigType = {
   activeLayout: 'vertical',
   showFileBrowser: true,
   layoutComponents: {
-    vertical: <VerticalLayout />,
-    horizontal: <HorizontalLayout />,
+    vertical: VerticalLayout,
```

Comment:
Outside scope of this PR...noting that on `weblab2`, the default is the vertical layout. But the default is still set in the layout toggle as horizontal.

Reply:
Do you mean the text "default"? Yeah I noticed that too, since web lab 2 is kind of on hold I'm doing minimal work on it (just keeping up with updates where it makes sense).

Reply:
>Do you mean the text "default"? 

Yes - thanks for filing the ticket!